crates/matrix-sdk/tests/integration/encryption/backups.rs

```diff
@@ -478,6 +484,302 @@ async fn steady_state_waiting() {
     server.verify().await;
 }
 
+async fn setup_create_room_and_send_message_mocks(server: &wiremock::MockServer) {
+    Mock::given(method("POST"))
+        .and(path("_matrix/client/unstable/room_keys/version"))
+        .and(header("authorization", "Bearer 1234"))
+        .respond_with(ResponseTemplate::new(200).set_body_json(json!({ "version": "1"})))
+        .mount(server)
+        .await;
+
+    Mock::given(method("POST"))
+        .and(path("_matrix/client/r0/createRoom"))
+        .and(header("authorization", "Bearer 1234"))
+        .respond_with(
+            ResponseTemplate::new(200)
+                .set_body_json(json!({ "room_id": "!sefiuhWgwghwWgh:localhost"})),
+        )
+        .mount(server)
+        .await;
+
+    let state = json!(
+        {
+            "algorithm": "m.megolm.v1.aes-sha2",
+            "rotation_period_ms": 604800000,
+            "rotation_period_msgs": 100
+        }
+    );
+
+    Mock::given(method("GET"))
+        .and(path("_matrix/client/r0/rooms/!sefiuhWgwghwWgh:localhost/state/m.room.encryption/"))
+        .and(header("authorization", "Bearer 1234"))
+        .respond_with(ResponseTemplate::new(200).set_body_json(state))
+        .mount(server)
+        .await;
+
+    Mock::given(method("GET"))
+    .and(path("_matrix/client/r0/user/@example:morpheus.localhost/account_data/m.secret_storage.default_key"))
+    .and(header("authorization", "Bearer 1234"))
+    .respond_with(
+        ResponseTemplate::new(404).set_body_json(json!({
+            "errcode": "M_NOT_FOUND",
+            "error": "Account data not found."
+        }))
+    )
+    .mount(server)
+    .await;
+
+    Mock::given(method("POST"))
+        .and(path("/_matrix/client/r0/keys/upload"))
+        .and(header("authorization", "Bearer 1234"))
+        .respond_with(ResponseTemplate::new(404).set_body_json(json!({
```

Comment:
It looks like a copy-pasta error: are you sure it's 404? 

@bnjbvr is fixing it in https://github.com/matrix-org/matrix-rust-sdk/pull/3953.